### ISSUE TITLE
% conversion to int

### ISSUE THREAD

**Issue by pjebs** [Sandertv/go-formula#6]
```go	
case token.REM:
		eval = func(vars vars) float64 {
			return float64(int(x(vars)) % int(y(vars)))
		}
	}
```

I was wondering what the rationale was for casting to int as currently implemented. I'm assuming it's because % only supports int arguments.

Perhaps a better option is https://golang.org/pkg/math/#Mod or https://golang.org/pkg/math/#Remainder (not sure what the difference is)

**Comment by Sandertv** [comment]
Definitely sounds off, I will change that to use math.Mod.

**Comment by Sandertv** [comment]
Resolved in b5818a30d019f040ca78c3cc82dc705b9f875fbd.

**Comment by pjebs** [comment]
Arguably it should be adjusted for v1 too.
